.gitignore

```diff
@@ -0,0 +1 @@
+properties.json
```

Comment:
I forgot to mention this in the PR overview. I moved `properties.json` to `properties.json.example`. Because it holds private user config it should be excluded from the repo.